docs/Setup/Demo.md

```diff
@@ -5,30 +5,65 @@ sidebar_position: 2
 
 The below steps would help you get a feel of Restel by running a demo server and test it with a pre-written excel file.
 
-## Pre-requisite
+## Cloning the git repository
 
+### Pre-requisite
+- JDK 11
 - Docker
 - [JsonBox](https://github.com/vasanthv/jsonbox)
 
-## Quickstart
+### Quickstart
 
```

Comment:
Quick start can be parent heading where it says, we can quick start either by cloning the repo or just by downloading the required files from the repo.